lib/ui/map/map_view.dart

```diff
@@ -78,6 +84,7 @@ class MapViewState extends State<MapView> {
           ),
           onMapCreated: (GoogleMapController controller) {
             _controller.complete(controller);
+            controller.setMapStyle(_mapStyle);
```

Comment:
This might create race conditions, if the map style was not loaded before onMapCreated is called.
```suggestion
            rootBundle.loadString('assets/map_style.txt').then((style) => controller.setMapStyle(style));
```